src/main/cli/cli.c

```diff
@@ -4140,6 +4141,10 @@ static bool prepareSave(void)
     }
 #endif
 
+#ifdef USE_TUNING_SLIDERS
+    applyTuningSliders(currentPidProfile);
```

Comment:
Not sure about this - this will mean that it is impossible for users that have the sliders enabled but have tweaked one or two individual parameters (which they can if using CMS or MSP) to create a `diff` and then restore their settings.
I think adding a dedicated `apply_sliders` or suchlike CLI command would be better.

Reply:
Added a separate command `apply_tuning_sliders`.